### DISCUSSION TITLE
If the exported csv file contains Chinese, unrecognized text will appear.

### DISCUSSION THREAD

**Discussion by xiaoyutx94** [datalust/seq-tickets#1740] [Help] [answered]
The exported csv file is UTF-8 encoded
![image](https://user-images.githubusercontent.com/5671609/211699653-59ecdfc5-c2a4-4e1d-acd6-316ca2ab63d4.png)

**Comment by nblumhardt** [comment, marked as answer]
Hi! Thanks for reaching out. I can't tell which characters/text are unexpected in your screenshot; would it be possible to highlight these, and also include (as text) the characters you expect? Thanks!

**Reply by xiaoyutx94** [reply]
![image](https://user-images.githubusercontent.com/5671609/211712085-66f9b995-6690-42bb-b154-faa4f138fd6b.png)
These are not real Chinese. The real content is "创建响应代码"
Thanks!

**Reply by liammclennan** [reply]
Thanks @snailg

I can't easily reproduce this behaviour. 

I tried logging `创建响应代码` and it exported to csv without a problem. 

<img width="822" alt="Screen Shot 2023-01-11 at 1 54 47 pm" src="https://user-images.githubusercontent.com/271514/211714386-08c3e3ee-af20-40c7-a886-1671f97af47e.png">

Which version of Seq are you using?

What do you see if you open the csv file as text?

**Reply by xiaoyutx94** [reply]
Thanks @liammclennan 
I am using windows msi 2022.1.7929.
maybe it's because I use office to open the csv file. when I use vscode or notepad to open the csv file, it's normal

**Reply by liammclennan** [reply]
Try choosing the 'UTF-8' option in the excel import text wizard.
